prov/efa/src/rxr/rxr.h

```diff
@@ -445,11 +445,14 @@ struct rxr_tx_entry {
 	uint16_t credit_allocated;
 
 	uint64_t total_len;
+	bool receipt_received;
 
 	enum rxr_tx_comm_type state;
 	struct rxr_queued_ctrl_info queued_ctrl;
 
 	uint64_t fi_flags;
+	uint16_t rxr_flags;
```

Comment:
Maybe just make it ```uint64_t```, otherwise compiler will add some padding space.

Reply:
Changed to `uint64_t`, resolving.